desc/compute/_basis_vectors.py

```diff
@@ -3764,3 +3988,247 @@ def _e_alpha_rp(params, transforms, profiles, data, **kwargs):
+        - data["e_theta_PEST"]
+        * (
+            data["theta_PEST_zz"]
+            - data["theta_PEST_tz"] * (data["theta_PEST_z"] / data["theta_PEST_t"])
+        )[:, jnp.newaxis]
+        - data["e_zeta_t"]
+        * (data["theta_PEST_z"] / data["theta_PEST_t"])[:, jnp.newaxis]
+    )
+
+    return data
+
+
+# TODO: Generalize for a general zeta before #568
+@register_compute_fun(
+    name="(e_theta_PEST_r)|PEST",
+    label="(\\partial_{\\rho} |_{\\phi, \\vartheta}"
+    " \\mathbf{e}_{\\vartheta}) |_{\\rho, \\phi}"
+    "= {\\mathbf{e}_{\\theta_{PEST}}}_{\\rho}",
+    units="m",
+    units_long="meters",
+    description="Derivative of the covariant poloidal PEST basis vector in (ρ,ϑ,ϕ)"
+    "coordinates or straight field line PEST coordinates w.r.t rho.",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=[
+        "e_theta_r",  # in DESC coordinates
+        "e_theta_PEST",
+        "(e_theta_PEST_v)|PEST",
+        "theta_PEST_t",
+        "theta_PEST_r",
+        "theta_PEST_rt",
+    ],
+    aliases=["(e_vartheta_r)|PEST", "(e_rho_v)|PEST"],
+)
+def _e_sub_vartheta_rz_rho_varthetaz(params, transforms, profiles, data, **kwargs):
+    data["(e_theta_PEST_r)|PEST"] = (
+        data["e_theta_r"]
+        - data["e_theta_PEST"] * (data["theta_PEST_rt"])[:, jnp.newaxis]
+        - data["(e_theta_PEST_v)|PEST"]
+        * (data["theta_PEST_r"] * data["theta_PEST_t"])[:, jnp.newaxis]
+    ) / data["theta_PEST_t"][:, jnp.newaxis]
+    return data
+
+
+# TODO: Generalize for a general zeta before #568
+@register_compute_fun(
+    name="(e_phi_r)|PEST",
+    label="\\partial_{\\rho} |_{\\phi, \\vartheta}"
+    " (\\mathbf{e}_{\\phi} |_{\\rho, \\vartheta})"
+    "= {\\mathbf{e}_{\\phi}}_{\\rho}",
+    units="m",
+    units_long="meters",
+    description="Derivative of the covariant toroidal basis vector in (ρ,ϑ,ϕ)"
+    "coordinates or straight field line PEST coordinates w.r.t rho."
+    "ϕ increases counterclockwise when viewed from above"
+    "(cylindrical R,ϕ plane with Z out of page).",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=[
+        "e_zeta_r",  # in native coordinates
+        "(e_phi_v)|PEST",
+        "(e_theta_PEST_v)|PEST",
+        "e_theta_PEST",
+        "(e_rho_v)|PEST",
+        "theta_PEST_r",
+        "theta_PEST_z",
+        "theta_PEST_rz",
+    ],
+    aliases=["(e_rho_p)|PEST"],
+)
+def _e_sub_phi_rvartheta_rho_varthetaz(params, transforms, profiles, data, **kwargs):
+    data["(e_phi_r)|PEST"] = (
+        data["e_zeta_r"]
+        - data["(e_phi_v)|PEST"] * data["theta_PEST_r"][:, jnp.newaxis]
+        - data["(e_theta_PEST_v)|PEST"]
+        * (data["theta_PEST_r"] * data["theta_PEST_z"])[:, jnp.newaxis]
+        - data["e_theta_PEST"] * (data["theta_PEST_rz"])[:, jnp.newaxis]
+        - data["(e_rho_v)|PEST"] * data["theta_PEST_z"][:, jnp.newaxis]
```

Comment:
This one is wrong. Need to delete line 4190, 4191, and 4193 and add the missing terms.